src/linter/ui5Types/fix/collections/sapUiCoreFixes.ts

```diff
@@ -18,6 +18,61 @@ t.declareModule("sap/ui/core/Configuration", [
 			moduleName: "sap/base/i18n/Localization",
 			mustNotUseReturnValue: true,
 		})),
+		t.method("getAccessibility", accessExpressionFix({
+			moduleName: "sap/ui/core/ControlBehavior",
+			propertyAccess: "isAccessibilityEnabled",
+		})),
+		t.method("getActiveTerminologies", accessExpressionFix({
+			moduleName: "sap/base/i18n/Localization",
+			propertyAccess: "getActiveTerminologies",
+		})),
+		t.method("getAllowlistService", accessExpressionFix({
+			moduleName: "sap/ui/security/Security",
+			propertyAccess: "getAllowlistService",
+		})),
+		t.method("getAnimation", callExpressionFix({
+			moduleName: "sap/ui/core/ControlBehavior",
+			propertyAccess: "getAnimationMode",
+			// Note: The new API returns an enum value instead of a boolean, therefore
+			// migration is currently not possible if the return value is used
+			// This could be optimized with an advanced migration that detects how the return
+			// value is used and e.g. migrates to something like
+			// (getAnimationMode() !== sap.ui.core.Configuration.AnimationMode.none)
+			// Right now this migration probably wont apply for most cases
+			mustNotUseReturnValue: true,
+		})),
+		t.method("getAnimationMode", accessExpressionFix({
+			scope: FixScope.FirstChild,
+			moduleName: "sap/ui/core/ControlBehavior",
+		})),
+		t.method("getCalendarType", accessExpressionFix({
+			moduleName: "sap/base/i18n/Formatting",
+			propertyAccess: "getCalendarType",
```

Comment:
The method name stays the same, so we can do this shortcut. This reduces the range of the change (i.e. code that we "touch") that will be generated for the fix, which is always preferable.

```suggestion
			scope: FixScope.FirstChild,
```